docker-compose.yml

```diff
@@ -1,50 +1,42 @@
-version: '3'
+version: '3.9'
 
 services:
     
   db:
     image: postgres
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
+      - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
     restart: always
-    env_file:
-      - .env
     environment:
-      POSTGRES_USER: ${DB_USERNAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
     ports:
       - 5432:5432
-    expose:
-      - "5432"
 
   api:
     build:
       context: .
-      dockerfile: Docker_rails
+      dockerfile: Dockerfile
```

Comment:
It's best to name your all your dockerfiles `Dockerfile` (it's just convention, but your IDEs will recognize it and give you some nice syntax highlighting)